recipe/meta.yaml

```
@@ -25,8 +25,11 @@ requirements:
     - click
     - tqdm
     - jinja2
-    - openff-toolkit-base >=0.11.0
+    - pydantic
+    - openff-toolkit-base >=0.14.0
     - openff-interchange-base
```

Just to safeguard against https://github.com/openforcefield/openff-interchange/issues/842, which toolkit 0.14 might pull in

```suggestion
    - openff-interchange-base >=0.3.18
```

---

ah nice - good catch!